RetailCoder.VBE/Navigation/CodeExplorer/CodeExplorerViewModel.cs

```diff
@@ -463,13 +466,16 @@ private void SwitchNodeState(CodeExplorerItemViewModel node, bool expandedState)
 
         public CommandBase ImportCommand { get; }
         public CommandBase ExportCommand { get; }
+        public CommandBase ExportAllCommand { get; }
+        public DelegateCommand _exportCommand { get; private set; }
```

Comment:
Properties should be `PascalCase`; if the command can/should only be assigned in the constructor, then if should be get-only/immutable - in which case the `private setter` needs to be dropped.